contracts/tokenomics/generator/src/contract.rs

```diff
@@ -1567,6 +1625,9 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> Result<Binary, ContractErro
     match msg {
         QueryMsg::ActivePoolLength {} => Ok(to_binary(&active_pool_length(deps)?)?),
         QueryMsg::PoolLength {} => Ok(to_binary(&pool_length(deps)?)?),
+        QueryMsg::EmissionRewards { lp_token, user } => {
+            Ok(to_binary(&query_emission_rewards(deps, lp_token, user)?)?)
```

Comment:
I think this value does not make sense. We need to know users' boost at the current period which is the value from the formula _min(0.4 + 0.6 * (w_i/W), 1)._ where w_i - user VP and W - total VP.

Reply:
Hm. It returns last calculated emission amount. If we'll use formula^ for calculate emission in real time, it will be wrong, because if we do nothing we have static emission amount calculated before and it changes only after next actions. Let's talk about it tomorrow.